.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,8 @@
 <!--
+Please only send in a PR to branches which are currently supported: https://laravel.com/docs/5.8/releases#support-policy 
+
+If you are unsure to which branch you want to send a PR please read: https://laravel.com/docs/5.8/contributions#which-branch
```

Comment:
```suggestion
If you are unsure to which branch you want to send a PR please read: https://laravel.com/docs/contributions#which-branch
```

Reply:
The docs don't support page forwards with anchor tags.